### PR TITLE
[Auctions-ish] Make ARSerifNav use a black status text with no black bg

### PR DESCRIPTION
**Pull request by orta**
This changes all our layouts, pushing them down by a status bar's worth of PX.

### CODE REVIEW

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionLotSetViewController.swift [hunk 1]
```diff
@@ -176,6 +176,12 @@ class LiveAuctionLotSetViewController: UIViewController {
 
     func dismissModal() {
         guard let presentor = splitViewController?.presentingViewController else { return }
+
+//        beginAppearanceTransition(false, animated: true)
+//        endAppearanceTransition()
+//
+
+
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionLotSetViewController.swift]: Should this be here?

**Comment by ashfurrow** [comment]
Looks cool, do you reckon the new status manager should have tests?

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
        
            2 Warnings
        
      </th>
     </tr>
  </thead>
  <tbody>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Needs testing on a Phone if change is non-trivial</td>
    </tr>
        <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
    
    
  </tbody>
</table>
  

  



Time | Class | Function |
| --- | ----- | ----- |286.8ms | [SwiftyJSON.swift](/artsy/eigen/tree/orta-auction_modals/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
282.0ms | [WebSocket.swift](/artsy/eigen/tree/orta-auction_modals/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
279.9ms | [AuctionViewController.swift](/artsy/eigen/tree/orta-auction_modals/Artsy/View_Controllers/Auction/AuctionViewController.swift#L181) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
91.1ms | [Observable.swift](/artsy/eigen/tree/orta-auction_modals/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
70.4ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/orta-auction_modals/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, sideMargin: String)




<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
Oh yeah! Time to use ForgeriesApplication

**Comment by orta** [comment]
🍏

**Pull request merged**